content/en/semi-assembly/assembling-frame/_index.md

```diff
@@ -0,0 +1,146 @@
+---
+title: "Assembling Frame"
+linkTitle: "Assembling Frame"
+weight: 10
+type: docs
+resources:
+- src: "**.png"
+- src: "**.jpg"
+description: >
+  Assembling the Frame
+---
+
+In this step, we'll assemble the main frame of the machine.
```

Comment:
Use second-person (you instead of we) when addressing the reader.

```markdown
In this step, you will assemble the main frame of the machine.
```

Alternatively:

```markdown
This guide walks you through assembling the main frame of the machine.
```